Adding tutorial files for project template demo blogs

merlin-quix opened the pull request:
Adding tutorial files for project template demo blogs

**Blog about Project template: image processing**
https://docs.google.com/document/d/13PkwLq4f7pgKyunOQpE9UDYBIw-DgyHD1q2Gp4YJLC4/edit?usp=sharing

**Blog about Project template: sentiment analysis**
https://docs.google.com/document/d/1o2Siw0_TACk3mT8Yw_-ax2o9acDcDTcaMuN-2OeHeD4/edit?usp=sharing

merltron-mc committed:
adding initial tutorial files to go with project templates

merltron-mc committed:
adding initial tutorial files to go with project templates

merltron-mc committed:
adding banned words

merltron-mc committed:
putting files into subfolders

merlin-quix committed:
fixing missing env var for input topic

merlin-quix committed:
adding latest tutorial changes

merlin-quix committed:
adding proper changes for tutorial

merlin-quix committed:
adding tutorial changes to room.service.ts

merlin-quix committed:
fixing web-chat.component.ts to be up to date with latest state

merlin-quix committed:
remove unnecessary state storage line  max-vehiclesv2_main.py

merlin-quix committed:
Update quix.service.ts

merlin-quix committed:
Update quix.service.ts

merlin-quix committed:
Update quix.service.ts

tbedford reviewed (approved):
I have tested the sentiment analysis code and it works fine. I did have to change workspaceId and generate my own PAT though. All good.

merlin-quix committed:
adding word boundary checking to quix_function.py

improved censorship

merlin-quix committed:
adding print output to max-vehiclesv2_main.py

merlin-quix committed:
Adding the "before" version of quix.service.ts

Adding the latest state of "quix.service.ts" so that readers can see what it looked like before we changed it for the tutorial.

merlin-quix committed:
Adding the "after" version of quix.service.ts

Adding the final version of the service (for the tutorial) to illustrate how it should look after the reader has made all the required changes.

merlin-quix committed:
Adding the "before" version (v2) of quix.service.ts

Adding the latest state of "quix.service.ts" so that readers can see what it looked like before we changed it for the tutorial.

merlin-quix committed:
Adding the "before" version (v2) of room.service.ts

Adding the latest state of "room.service.ts" so that readers can see what it looked like before we changed it for the tutorial.

merlin-quix committed:
Adding the "before" version (v2) of web-chat.component.ts

Adding the latest state of "web-chat.component.ts" so that readers can see what it looked like before we changed it for the tutorial.

merlin-quix committed:
Adding the "after" version (v2) of quix.service.ts 

Adding the final version of the service (for the tutorial) to illustrate how it should look after the reader has made all the required changes.

merlin-quix committed:
Adding the "after" version (v2) of room.service.ts

Adding the final version of the service (for the tutorial) to illustrate how it should look after the reader has made all the required changes.

merlin-quix committed:
Adding the "after" version (v2) of web-chat.component.ts

Adding the final version of the service (for the tutorial) to illustrate how it should look after the reader has made all the required changes.
